### PR TITLE
feat(makefile): move apply-services into makefile

### PR DESCRIPTION
**Pull request by dagoof**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,16 +1,22 @@
...
-DEPLOY_ENV ?= notifications
-CF_API     ?= https://api.system.$(CLD_HOST)
-CF_ORG     ?= dta
-CF_SPACE   ?= $(DEPLOY_ENV)
-CF_HOME    ?= $(HOME)
+CLD_HOST    ?= y.cld.gov.au
+DEPLOY_ENV  ?= notifications
+CF_API      ?= https://api.system.$(CLD_HOST)
+CF_ORG      ?= dta
+CF_SPACE    ?= $(DEPLOY_ENV)
```
Comment (jonathaningram) [inline Makefile]: @dagoof you can remove `DEPLOY_ENV` and just make `CF_SPACE= notifications`. Unless you were planning on using `DEPLOY_ENV` for other things later?

File: ci/apply-services [hunk 2]
```diff
@@ -1,24 +0,0 @@
-#!/usr/bin/env bash
```
Comment (jonathaningram) [inline ci/apply-services]: @dagoof you happy to keep the .sample.json files in or is there a better way?

Comment (dagoof) [reply ci/apply-services]: i want to leave them in. we need a better solution eventually but they're pretty crucial right now so we at least know what should exist

File: Makefile (hunk 1 above)
Comment (dagoof) [reply Makefile]: I think I wanted to decouple our notion of environment /app name from the deploy mechanism. but I don't know if it even matters anymore

**Pull request merged**